airbyte-integrations/connectors/source-woocommerce/build.gradle

```diff
@@ -0,0 +1,20 @@
+plugins {
+    id 'airbyte-python'
+    id 'airbyte-docker'
+    id 'airbyte-source-acceptance-test'
+}
+
+airbytePython {
+    moduleDirectory 'source_woocommerce'
+}
+
+dependencies {
+    implementation files(project(':airbyte-integrations:bases:source-acceptance-test').airbyteDocker.outputs)
+    implementation files(project(':airbyte-integrations:bases:base-python').airbyteDocker.outputs)
+}
+
+// used to allow local iteration to work
+task('installSingerTap', type: PythonTask) {
+    module = "pip"
+    command = "install woocommerce"
+}
```

Comment:
nit: end of line

Reply:
Hi Marcos, I have to remove the `install woocommerce` , its not more needed. I see your comments however they are not explicit enough for me. Could you please add more details please.

Reply:
Please add new line after last character.